json-schema/nestjs-goldenpath/config/application/ecs-application-config.schema.json

```
@@ -1,5 +1,5 @@
 {
-  "$id": "https://raw.githubusercontent.com/ageras-com/schema-collection/v1/json-schema/nestjs-goldenpath/config/application/ecs-application-config.schema.json#",
+  "$id": "https://raw.githubusercontent.com/ageras-com/schema-collection/feat-config-add-migrations-db-schema/json-schema/nestjs-goldenpath/config/application/ecs-application-config.schema.json#",
```

For all the occurences.
```suggestion
  "$id": "https://raw.githubusercontent.com/ageras-com/schema-collection/v1/json-schema/nestjs-goldenpath/config/application/ecs-application-config.schema.json#",
```